### DISCUSSION TITLE
Which emoji database is used?

### DISCUSSION THREAD

**Discussion by wleoncio** [marp-team/marp#44] [Q&A] [answered]
I am making a Marp presentation containing several country flags.

As a general rule, I've been using `:country:` to display a flag, where "`country`" is taken as the "GitHub" entry for a certain country at https://emojipedia.org/.

This works for the most part, but for some countries it doesn't. In particular, I cannot find a way to display sub-UK flags (England, Scotland, Wales) other thank copy-pasting the code from Emojipedia itself, which is not ideal.

It seems that the emoji style used is the one from [Twitter](https://emojipedia.org/twitter/), which makes me think it should be able to display these flags.

How do I display these flags using the `:country:` format, then?

**Comment by yhatt** [comment, marked as answer]
Marp is using a plugin based on [markdown-it-emoji](https://github.com/markdown-it/markdown-it-emoji) v1.4.0 to render emoji shorthands. [The list of shorthands is here.](https://github.com/markdown-it/markdown-it-emoji/blob/1.4.0/lib/data/full.json)
https://github.com/marp-team/marp-core/blob/master/src/emoji/emoji.ts

markdown-it-emoji had not updated in the last 3 years, but recently [it has updated into v2 a month ago](https://github.com/markdown-it/markdown-it-emoji/blob/master/CHANGELOG.md#200--2020-10-15), including updated emoji DB. Pull request for bumping markdown-it-emoji in https://github.com/marp-team/marp-core is welcome.

**Reply by yhatt** [reply]
**UPDATE**: [Marp Core v1.4.0](https://github.com/marp-team/marp-core/releases/tag/v1.4.0) has included an updated emoji DB from [markdown-it-emoji v2.0.0](https://github.com/markdown-it/markdown-it-emoji/blob/2.0.0/lib/data/full.json). Marp tools depending on Marp Core v1.4.0 can use extra 358 shorthands for emoji.
